### DISCUSSION TITLE
v1.3

### DISCUSSION THREAD

**Discussion by xabaras** [xabaras/RecyclerViewSwipeDecorator#17] [General]
With this version RecyclerViewSwipeDecorator moves to AndroidX breaking backward compatibility.

<hr /><em>This discussion was created from the release <a href='https://github.com/xabaras/RecyclerViewSwipeDecorator/releases/tag/1.3'>v1.3</a>.</em>

**Comment by mbalderas** [comment]
Do you have any plans to port this to Xamarin.Android or any type of C#?
